app/models/banana.rb

```diff
@@ -0,0 +1,12 @@
+class Banana < ActiveRecord::Base
+  def cut
+    "cut cut!"
```

Comment:
__[staging][rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols.